pipelines/release/official_release.groovy

```diff
@@ -184,7 +187,7 @@ notify.wrap {
             DOCKER_TAGS: extraDockerTags,
             MANIFEST_ID: manifestId,
             LSST_COMPILER: lsstswConfig.compiler,
-            SPLENV_REF:splenvRef,
```

Comment:
This changes to rubinEnvVer rather than adding that as another parm?

Reply:
Right.  Only one parameter is needed to select the base newinstall container, so I left it as SPLENV_REF.  The value that is passed in is the rubin-env version rather than the job's `splenvRef`, which might be an eups tag.